ci/integration.yml

```diff
@@ -0,0 +1,31 @@
+---
+resources:
+  - name: secrets
+    type: s3-iam
+    source:
+      bucket: {{state_bucket}}
```

Comment:
The `run-tests` task returns an error:
```
failed to load pr/ci/integration.yml: yaml: invalid map key: map[interface {}]interface {}{"state_bucket":interface {}(nil)}
```
I don't think variable interpolation works for external tasks and that this resource definition should be moved to `alphagov/paas-release-ci` where we define the outer pipeline and these variables.